Tensile/SolutionStructs.py

```diff
@@ -2469,8 +2478,15 @@ def assignDerivedParameters(state):
     ########################################
     while True: # exit criteria at end
       validDepthU = True
+      # peek LoopIters
+      loopIters = (depthU // state["LocalSplitU"]) // state["InnerUnroll"]
+      if "MatrixInstK" in state:
```

Comment:
For consistency: EnableMatrixInstruction

Reply:
done

Reply:
EnableMatrixInstruction will always be in state (true or false). You just need:

```suggestion
      if state["EnableMatrixInstruction"]:
```